src/main/java/io/dropwizard/primer/auth/filter/PrimerAuthAnnotationFilter.java

```diff
@@ -63,15 +63,8 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
 
                 // Execute authorizer
                 if (authorizer != null && !authorizer.authorize(webToken, requestContext, authorize)) {
-                    handleException(
-                            PrimerException.builder()
-                                    .errorCode("PR004")
-                                    .message("Unauthorized")
-                                    .status(401)
-                                    .build(),
-                            requestContext,
-                            token.get()
-                    );
+                    // Abort request without blacklisting the token. FIXME: This should be controlled by the authorizer.
+                    abortRequest(requestContext, Response.Status.FORBIDDEN, PrimerError.builder().errorCode("PR002").message("Forbidden").build());
```

Comment:
@phaneesh Do you think if the authorizer fails, should it return 401 or 403?

Reply:
@phaneesh The reason I had initially put 403 was because we do not want to blacklist the token for 403. Here we are just aborting the request without blacklisting.
Throwing PrimerException will blacklist the token in the `handleError` block.
I'll think of a different design for doing this and discuss with you.